### DISCUSSION TITLE
question regarding mass unliker and internet footprint

### DISCUSSION THREAD

**Discussion by Kholekardashian** [AprilSylph/XKit-Rewritten#1631] [Help Desk]
Hi again! I have previously posted a question a few months back regarding an issue I had with mass unliker (resolved now) and had another inquiry about said feature. My personal use for mass unliker was primarily to combat my internet footprint on Tumblr, however I find I am still able to find evidence of my likes in things like google search. For example, searching "[Tumblr URL] Tumblr" into google can bring up results saying "liked by [URL]". My blog settings via Tumblr also limit 3rd party search engine results, however I understand that is an imperfect setting. My question is, why is that the case, and what can I do to resolve that issue (if anything)? Thankful for any insight, and thanks again to the people who helped me previously!

**Comment by marcustyphoon** [comment]
I don't personally know the answer to this! Google's cache definitely takes a while to update, so it may just be a waiting game.
